docs/Developers/Writing_Good_Commits.rst

```diff
@@ -6,42 +6,45 @@ Quick reference
 
 Example of a good commit:
 
+We are looking at pull request `#5759 <https://github.com/coala/coala/pull/5759>`_
+
 ::
 
-    setup.py: Change bears' entrypoint
+    Haskell.py: Add Haskell.py file
 
-    This entrypoint ensures that coala discovers
-    the bears correctly.
-    It helps not writing more functions inside
-    ``coalib`` for this.
+    Add a new python file in
+    coalib/bearlib/languages/definitions/Haskell.py
 
-    Closes https://github.com/coala/coala/issues/5861
+    Closes https://github.com/coala/coala/issues/5330
 
-- `setup.py: Change bears' entrypoint`: Describe the change in
+- `Haskell.py: Add Haskell.py file`: Describe the change in
    maximum of 50 characters.
 
-- `This entrypoint.. ..for this`: Describe the reasoning of your changes
-   in maximum of 72 characters per line.
+- `Add a new python file in.. ..Haskell.py`: Describe the reasoning
+   of your changes in maximum of 72 characters per line.
 
-- `Closes https://github.com/coala/coala/issues/5861`: Mention the URL
+- `Closes https://github.com/coala/coala/issues/5330`: Mention the URL
    of the issue it closes or fixes.
 
+We are now looking at pull request `#5789 <https://github.com/coala/coala/pull/5789>`_
+
 ::
 
-    configure.py: Fix docstring typo
+    ConfParserTest.py: Fix typos in comments
 
-    This fixes the typo and changes
-    it from wether --> whether.
+    This fixes multiple occurences of typos in the comments
+    omment --> comment
+    inexistent --> non-existent
 
-    Closes https://github.com/coala/coala/issues/4018
+    Fixes https://github.com/coala/coala/issues/5785
```

Comment:
This change wasnt fixing a bug, and he changed it to be a Closes during the code review .

See https://github.com/coala/coala/pull/5789/commits

You can still use this one here, if you use the final commit message which is Closes.

You could then mention that this is an example where the shortlog can use "fix", which isnt a "Fixes" because nothing was broken and it didnt have a bug label.  Or maybe it is too confusing ;-)

https://github.com/coala/coala/pull/5544 is a funny example of a `Fixes` .  As TrumpScript is a fictional language, it shouldnt use an file extension which is used by real language TypeScript.  It was causing real bugs.

Reply:
Yes I thought so too but his initial PR is writing "fixes" so I thought it will confuse if I change it to "closes"?
![screenshot_2018-10-21 confparsertest py fix typos in comments by haxzie pull request 5789 coala coala](https://user-images.githubusercontent.com/6190813/47267345-2ed87000-d54b-11e8-9a90-f22a4e6a4b32.png)

Do you think I can provide the link you gave me instead ie. https://github.com/coala/coala/pull/5789/commits/87a809336adcb79a0b5b370ccdf0ba9b34213ec8 ?

 I thought that if I noted "closes" a newbie might get confused from the initial PR message saying fixes (which was not edited) that's why I left it like that.

So should I point to the above url and use "closes" instead?

Reply:
Ya, good thinking.  I have edited https://github.com/coala/coala/pull/5789 , so it now says `Closes`.